src/views/Home/Home.test.jsx

```diff
@@ -11,5 +14,11 @@ const user = {
 }
 
 test('Should render the user profile', () => {
-
+  render(<Profile user={user} />)
+  screen.findByText('Vonta')
+  screen.findByText('Res Non Verba')
+  screen.findByText('Interests')
+  screen.findByText('React')
```

Comment:
Don't forget to test for the list of user likes here. We can do something like const list = await screen.findByRole('listitem'). Then our expect statement will test for the length of the list items.